index.js

```diff
@@ -45,6 +46,34 @@ function filterResponseForImportantEvents(allEventsFromFetch) {
     return arrayOfImportantEvents;
 }
 
+function shouldIncludeEvent(eventType, association = 'OWNER') {
```

Comment:
It looks like this function is never called with an association parameter. Should we just remove the parameter, and hard-code 'OWNER' in the if statement? It would make the logic much easier to parse.